client/src/components/Images.tsx

```diff
@@ -0,0 +1,33 @@
+//@ts-ignore
+import brick from '../assets/backgrounds/amanda-jones-dNyvA2i97uQ-unsplash.jpg';
+//@ts-ignore
+import cat from '../assets/backgrounds/bantersnaps-zGiST_eUAXo-unsplash.jpg';
+//@ts-ignore
+import fish from '../assets/backgrounds/brooke-lark-iXhfpWEnsN0-unsplash.jpg';
+//@ts-ignore
+import ship from '../assets/backgrounds/bryan-goff-x02dRo6PEIY-unsplash.jpg';
+//@ts-ignore
+import tree from '../assets/backgrounds/david-hofmann--Ae2HFODn2Q-unsplash.jpg';
+//@ts-ignore
+import bus from '../assets/backgrounds/eggbank-8qb4dBHkA7g-unsplash.jpg';
+//@ts-ignore
+import astronaut from '../assets/backgrounds/jordan-mcgee-Ev4U6ieA4zA-unsplash.jpg';
+//@ts-ignore
+import architecture from '../assets/backgrounds/logan-weaver-e_galhGvUJ8-unsplash.jpg';
+//@ts-ignore
+import plant from '../assets/backgrounds/scott-webb-BLBCj6dxaSE-unsplash.jpg';
+//@ts-ignore
+import galaxy from '../assets/backgrounds/solen-feyissa-KziPJwmo91A-unsplash.jpg';
+
+export const images = [
+    { key: "brick", src: brick },
+    { key: "cat", src: cat },
+    { key: "fish", src: fish },
+    { key: "ship", src: ship },
+    { key: "tree", src: tree },
+    { key: "bus", src: bus },
+    { key: "astronaut", src: astronaut },
+    { key: "architecture", src: architecture },
+    { key: "plant", src: plant },
+    { key: "galaxy", src: galaxy }
+];
```

Comment:
this is redundant, App.tsx should import this images variable instead of having its own copy